File: INICIANTE/1046/main.c
#include <stdio.h>

int elapsed(int start, int finish) {
    if (finish > start) return finish - start;
    return finish - start + 24;
}

int main() {
    int start, finish;
    scanf("%d %d", &start, &finish);
    printf("O JOGO DUROU %d HORA(S)\n", elapsed(start, finish));
    return 0;
}

File: INICIANTE/1046/main.dart
import 'dart:io';

int elapsed(int start, int finish) {
  if (finish > start) return finish - start;
  return finish - start + 24;
}

void main() {
  var [start, finish] = stdin.readLineSync()!.split(' ').map(int.parse).toList();
  print('O JOGO DUROU ${elapsed(start, finish)} HORA(S)');
}
